source/_patterns/01-elements/select/select.scss

```diff
@@ -96,6 +101,8 @@
 
 	// Floating labels
 	& + .elm-label {
+		@extend %form-label;
```

Comment:
It looks wrong when there is no option and no label...
Can we provide a nonempty option to be selected?

![image](https://github.com/db-ui/core/assets/104347736/17e30c78-a07b-4bbb-ae42-cbe87ce37256)

Reply:
Of course, that is very easy in patternlab.